### PR TITLE
Migrate `tiptap` to v2

### PR DESCRIPTION
**Pull request by Rotzbua**
Tiptap v2 is out. Now with typescript support.

Migration guide:
https://tiptap.dev/overview/upgrade-guide

They changed the extension system, so a lot of packages are required.

### CODE REVIEW

File: src/components/admin/AdminNewsForLauncher.vue [hunk 1]
```diff
@@ -207,38 +202,53 @@
...
+import { Blockquote } from "@tiptap/extension-blockquote";
+import { Image } from "@tiptap/extension-image";
+import { BulletList } from "@tiptap/extension-bullet-list";
+import { ListItem } from "@tiptap/extension-list-item";
+import { Heading } from "@tiptap/extension-heading";
+import { HardBreak } from "@tiptap/extension-hard-break";
+import { CodeBlock } from "@tiptap/extension-code-block";
+import { HorizontalRule } from "@tiptap/extension-horizontal-rule";
+import { OrderedList } from "@tiptap/extension-ordered-list";
+import { TaskList } from "@tiptap/extension-task-list";
```
Comment (gustav87) [inline src/components/admin/AdminNewsForLauncher.vue:247]: Are we using TaskList? Do we need the import? Same question goes for the rest of the imports.

Comment (gustav87) [reply src/components/admin/AdminNewsForLauncher.vue:247]: I don't see it in the menubar, hence my question :)

Comment (gustav87) [reply src/components/admin/AdminNewsForLauncher.vue:247]: Don't worry about it if you don't know. We can fix it later, too.

Comment (Rotzbua) [reply src/components/admin/AdminNewsForLauncher.vue:247]: They renamed `Todo*` to `Task*`
It was used before here:
https://github.com/w3champions/website/pull/680/files#diff-eda8fafd78e785a9c6b4b2873a29cac23ec7186e729933ca071f1799cdbfe48cL221-L222
But not sure if it was really used 🤔

Comment (gustav87) [reply src/components/admin/AdminNewsForLauncher.vue:247]: Could we import only the extensions we use? Or we can remove them later, but might as well do it now!

**Comment by gustav87** [comment]
I'm gonna merge this in and we'll clean it up in another pr! Thanks for the contribution. Awesome stuff :)

**Comment by gustav87** [comment]
Heya, I can't request your review because you're not part of the organization. I'll invite you. You're doing a lot of good work so why not! :)

**Comment by gustav87** [comment]
Hey @Rotzbua, don't know where to contact you so I'll write here. You're welcome to join us developing for our new Launcher app, written in React in the frontend and Rust in the backend. I know you've done great things in the website repo, so if you wanna join us, shoot me a message :)

**Pull request merged**